### DISCUSSION TITLE
Error Transferring Repo

### DISCUSSION THREAD

**Discussion by chevan-nanayakkara** [community/community#124842] [Repositories] [answered]
### Select Topic Area

Question

### Body

I'm trying to transfer a repo from one personal account to another. After I initiate and approve the transfer I'm receiving an error message and I end up having to abort the transfer. Looking for some help.

**Comment by shivamgaur99** [comment, marked as answer]
you can follow this step

1. Check Permissions: Make sure you have the right permissions on both accounts.

2. Verify Ownership: Ensure you own the repository you're trying to transfer.

3. Follow Correct Steps: Initiate and approve the transfer correctly.

4. Note Error Message: Pay attention to the specific error you're getting.

5. Seek GitHub Support: If all else fails, reach out to GitHub support for help.

if you find this helpful pleasae marked as an answer

**Reply by sadisky** [reply]
transactions error

**Reply by khadijamuhammadnadeem499-cpu** [reply]
after all these error is not resolved
